content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -0,0 +1,135 @@
+
+1. Update the Helm repository:
+
+    {{< text syntax=bash snip_id=update_helm >}}
+    $ helm repo update istio
+    {{< /text >}}
+
+## In-place upgrade
+
+You can perform an in place upgrade of Istio in your cluster using the Helm
+upgrade workflow.
+
+Before upgrading Istio, it is recommended to run the `istioctl x precheck` command to make sure the upgrade is compatible with your environment.
+
+{{< text syntax=bash snip_id=istioctl_precheck >}}
+$ istioctl x precheck
+✔ No issues found when checking the cluster. Istio is safe to install or upgrade!
+  To get started, check out <https://istio.io/latest/docs/setup/getting-started/>
+{{< /text >}}
+
+{{< warning >}}
+[Helm does not upgrade or delete CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) when performing an upgrade. Because of this restriction, an additional step is required when upgrading Istio with Helm.
+{{< /warning >}}
+
+### Manually upgrading the CRDs and Istio base chart
+
+1. Upgrade the Kubernetes custom resource definitions ({{< gloss >}}CRDs{{</ gloss >}}):
+
+    {{< text syntax=bash snip_id=manual_crd_upgrade >}}
+    $ kubectl apply -f manifests/charts/base/crds
+    {{< /text >}}
+
+1. Upgrade the Istio base chart:
+
+    {{< text syntax=bash snip_id=upgrade_base >}}
+    $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
+    {{< /text >}}
+
+### Upgrading the Istio discovery Component
+
+Istiod is the control plane component that manages and configures the proxies to route traffic within the ambient mesh.
+
+{{< text syntax=bash snip_id=upgrade_istiod >}}
+$ helm upgrade istiod istio/istiod -n istio-system
+{{< /text >}}
+
+### Upgrading the ztunnel component
+
+The ztunnel DaemonSet is the L4 node-proxy component of ambient.
+{{< warning >}}
+Upgrading ztunnel in-place will briefly disrupt all ambient mesh traffic on the node.
+{{< /warning >}}
+
+{{< text syntax=bash snip_id=upgrade_ztunnel >}}
+$ helm upgrade ztunnel istio/ztunnel -n istio-system
+{{< /text >}}
+
+### (Optional) Upgrading the Gateway component
+
+Gateway components manage east-west and north-south dataplane traffic between ambient mesh boundaries, as well as some aspects of the L7 dataplane.
+
+{{< text syntax=bash snip_id=upgrade_gateway >}}
+$ helm upgrade istio-ingress istio/gateway -n istio-ingress
+{{< /text >}}
+
+### (Optional) Upgrading the CNI Component
+
+The Istio CNI agent is responsible for detecting the pods that belong to the ambient mesh, and configuring the traffic redirection between pods and the ztunnel DaemonSet. It is not part of the data plane or control plane, and so does not need to be upgraded on the same cadence as the control plane and data plane.
+
+// TODO BML - so, on what cadence SHOULD it be upgraded, if it should not be upgraded with the rest of Istio?
```

Comment:
Might be worth referencing https://istio.io/latest/docs/setup/additional-setup/cni/#upgrade, it can be upgraded in any order provided it is within 1 minor version of the control plane. So while we don't say before/after, this could be different for ambient.

Reply:
I'll copy the wording around the version constraint as-is, ty!